User/models/UserModel.ts

```diff
@@ -0,0 +1,41 @@
+import { Model, DataTypes } from 'sequelize';
+import database from '../../config/database';
+
+export class UserModel extends Model {
+    public id!: number;
+    public login!: string;
+    public password!: string;
+    public age!: number;
+	public isDeleted!: boolean;
+}
+
+UserModel.init ({
+	id: {
+		type: DataTypes.STRING,
+		autoIncrement: true,
+		primaryKey: true,
+	},
+	login: {
+		type: DataTypes.STRING,
+		allowNull: false,
+		unique: true,
+	},
+	password: {
+		type: DataTypes.STRING,
+		allowNull: false,
+	},
+	age: {
+		type: DataTypes.INTEGER,
+		allowNull: true,
+	},
+	isDeleted: {
```

Comment:
There is some common practice
![image](https://user-images.githubusercontent.com/17451948/73384310-102f6000-42dc-11ea-9951-8564c371e6ce.png)

![image](https://user-images.githubusercontent.com/17451948/73384180-ce061e80-42db-11ea-86f8-8543fe38fae6.png)

So, you can remove your custom isDeleted flag)) You can easily distinguish deleted users by timestamp. So, if NULL, it means that record is not deleted.